js/plugins/Header.jsx

```diff
@@ -8,21 +8,39 @@
 import {useEffect} from "react";
 import { createPlugin, connect } from "@mapstore/utils/PluginsUtils";
 
-export const Header = ({url = "/header/", page = "mapstore", height = 90, ignoreIFrame = false}) => {
+export const Header = ({url = "/header/", page = "mapstore", height = 90, ignoreIFrame = false, script = "https://cdn.jsdelivr.net/gh/georchestra/header@dist/header.js", legacy = false}) => {
     useEffect(() => {
         const header = document.getElementById("georchestra-header");
         const container = document.getElementById("container");
         if (header) {
+            // Prevent creation of multiple headers
+            const existingGeorHeader = document.getElementsByTagName("geor-header");
+            if (existingGeorHeader?.length > 0) {
+                existingGeorHeader[0].remove();
+            }
+            const existingScript = document.querySelectorAll("script[src='" + script + "']");
+            if (existingScript?.length > 0) {
+                existingScript[0].remove();
+            }
             if (!ignoreIFrame && window.location !== window.parent.location) {
                 header.style.display = 'none';
                 if (container) {
                     container.style.top = '0';
                 }
             } else {
-                header.style.display = 'block';
-                header.src = url + "?active=" + page;
+                const georHeader = document.createElement("geor-header");
+                const georHeaderScript = document.createElement("script");
```

Comment:
Just wondering if we couldn't put these elements in the `index.html` and then just set the script.src and attributes here,... also to prevent the check for multiple header creation above. But I guess this does not work?

Reply:
I modify it :)